bin/main.rb

```diff
@@ -1,2 +1,32 @@
-#!/usr/bin/main ruby
-puts 'Hello World'
+#!/usr/bin/env ruby
+def create_player
+  puts 'Please enter name of the first player'
+  player1 = gets.chomp
+  puts "Welcome #{player1}"
+  puts 'Please enter name of the second player'
+  player2 = gets.chomp
+  puts "Welcome #{player2}"
+end
+create_player
+
+def player_key
+  puts ' Choose your key'
+  key1 = gets.chomp
+  puts "Thanks for choosing #{key1}"
+  puts ' Choose your key'
+  key2 = gets.chomp
+  puts "Thanks for choosing #{key2}"
+end
+player_key
```

Comment:
The flow of logic here isnt great. It does not have to work 100% but it does have to have good logical flow.
As it stands one does not know which player is supposed to enter key. 

Maybe ask for key in context of player or optionally, consider having them choose between x and o to limit. (This can be a comment as well)